lib/generators/rails_template18f/terraform/templates/terraform/README.md.tt

```diff
@@ -87,47 +80,59 @@ The below steps rely on you first configuring access to the Terraform state in s
 
     The easiest way to use this script locally is to redirect the output directly to the `secrets.auto.tfvars` file it needs to be used in
 
+1. Create a new `sandbox-name.tfvars` file to hold variable values for your environment.
+
 1. Run terraform from your new environment directory with
     ```bash
-    terraform init -backend-config="profile=<%= app_name %>-terraform-backend"
-    terraform plan
+    terraform plan -var-file=sandbox-name.tfvars -var rails_master_key=$(cat ../config/master.key)
```

Comment:
[question (nb)] should there be something that writes the options to `sb-name.tfvars`?